libr/socket/socket.c

```diff
@@ -311,7 +311,7 @@ R_API bool r_socket_connect(RSocket *s, const char *host, const char *port, int
 					s->fd = -1;
 					continue;
 				}
-				r_socket_block_time (s, 0, 0, 0);
+				r_socket_block_time (s, 10000, 1000, 0);
```

Comment:
i would like to have some defines to specify some default timeout values here, (and reuse them in the other cases below). which time do you think it shuold be fine to define for waiting until the connection succeds?

from internet:

* In Windows the value is dynamic for established conections, though the default for initial connections is 72 seconds. 

imho thats too much, i would go for having lower timeouts like 5s

Reply:
No real preference here. 5s seems like a good random value for now!

Reply:
Don't aprove it until the implementation is done. you can review it again now